modules/odc-creating-functions.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-functions.adoc
+
+:_content-type: PROCEDURE
+[id="odc-creating-functions_{context}"]
+= Creating a function in the web console
+
+You can create a function from a Git repository by using the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have logged in to the {product-title} web console.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have created or have access to a Git repository that contains the code for your function. The repostory must contain a `func.yaml` file or use the `s2i` build strategy.
+
```

Comment:
This procedure also requires that the Tasks and Pipeline for building functions on the cluster be installed. The task installation has previously been documented in the section for building functions on the cluster [here](https://docs.openshift.com/container-platform/4.12/serverless/functions/serverless-functions-on-cluster-builds.html#serverless-functions-creating-on-cluster-builds_serverless-functions-on-cluster-builds).

We are working with the Pipelines team to ensure that these Tasks, and the new Pipeline resource will be installed through artifacthub.io but the URLs for those are not yet available.